Makefile

```diff
@@ -0,0 +1,28 @@
+GO ?= go
+GOLANGCI_LINT ?= golangci-lint
+GOLANGCI_LINT_VERSION ?= v1.29.0
```

Comment:
Maybe to set the currently latest version https://github.com/golangci/golangci-lint/releases/tag/v1.30.0?